src/app/View.elm

```diff
@@ -47,4 +54,4 @@ tableView players cup =
         playersTuples =
             List.map2 idAndPlayerTo3Tuple ids players
     in
-        Table.view Types.TableAction <| Table.initWithPlayersAndCup playersTuples cup
+        Table.view Types.TableAction <| Table.fromPlayersAndCup playersTuples cup
```

Comment:
`Table.view` currently doesn't allow it to be told which one is selected. Will add this later.